projects/cdk/abstract/dialog.service.ts

```diff
@@ -23,8 +23,8 @@ export abstract class AbstractTuiDialogService<T> extends Observable<
         super(observer => this.dialogs$.subscribe(observer));
     }
 
-    open<G>(
-        content: PolymorpheusContent<TuiBaseDialogContext<G> & T>,
+    open<G = void>(
+        content: PolymorpheusContent<TuiAlertContext<T, G>>,
```

Comment:
This is a common abstract class for both dialogs and alerts, please revert this interface change.

Reply:
fixed